scripts/pipeline.lua

```diff
@@ -1,21 +1,27 @@
 local pipelineDepth = 1
 local counter = 0
 local maxRequests = -1
+local method = "GET"
 
 function init(args)
+
```

Comment:
Spoke offline, the new scenario needs both pipelining and methods. Will keep the changes to the script but will change when the script is used by wrk.